### DISCUSSION TITLE
Maximum trainable size

### DISCUSSION THREAD

**Discussion by mlfffinder** [ACEsuit/mace#150] [Q&A] [answered]
Hi
I have some questions.
1. Considering the memory usage or training hours, what is the practically maximum trainable number of atoms in the cell?  

2. Is multi-gpu training available? 

Thanks.

**Comment by ilyes319** [comment]
Hey!
Thank you for your questions and your interest in MACE.

1. A few hundred (300 max) atoms per cell is the maximum I would go. Note that MACE is a local model, and considering larger cells than that would not provide obvious advantages. Moreover, it would force you to use a less accurate electronic structure level of theory for generating the data, which I do not recommend.

2. Yes multi-gpu is available in the [multi-gpu branch](https://github.com/ACEsuit/mace/tree/multi-GPU). There is a documentation on how to use it in the readme. It will be merged soon.

**Comment by gabor1** [comment, marked as answer]
What @ilyes319 writes is how we have been training ML force fields up until now - i.e. 10-100-300 atom unit cells, and as many of these as necessary or available. But I am quite interested in training models on many fewer but much larger cells. For example, are a few special aspects of this training: you will have very few energy values, but the same number of force components as with many cells. What is your use case for having a large number of cells? 

Combining large cells with small cells certainly would be straightforward. So if you have lots of small cells, and a few large ones that have special structures in them, I can't see any problem with that.

**Reply by mlfffinder** [reply]
In my case, I have some slab model of heterostructure which could't be made in small cell but it is ~300 atom and yeah they are only few and majorities are ~100 atoms.  I'll try it, thx for the reply

**Reply by gabor1** [reply]
That's still "small" in my book. As long as you have forces.
